src/main/java/slimeknights/tconstruct/library/recipe/tinkerstation/ITinkerStationRecipe.java

```diff
@@ -30,7 +34,17 @@ default ItemStack getCraftingResult(ITinkerStationInventory inv) {
     return getRecipeOutput().copy();
   }
 
-  default int getAmountUsed(int index) {
-    return 1;
+  /**
+   * Handles subtracting the inputs used by the recipe and passes any extra items (container items) to the consumer
+   *
+   * @param inventory the actual items in the inventory
+   * @param extraStackConsumer handles the extra stacks if they are unable to be added to the inventory
+   */
+  void consumeInputs(List<ItemStack> inventory, Consumer<ItemStack> extraStackConsumer);
+
+  /** Unused, please call consumeInputs */
```

Comment:
I typically would write this as `@deprecated use {@link #consumeInputs(List,Consumer)}` so you get the fancy javadoc formatting